tests/test_base_device.py

```diff
@@ -180,38 +179,6 @@ def test_no_shots_warns_and_defaults(self):
 
         assert dev.shots.total_shots == 1024
 
-    @pytest.mark.skip(reason="Options handling not decided on yet")
-    def test_kwargs_on_initialization(self, mocker):
```

Comment:
This is gone since we don't have an Options() object anymore. It is replaced by a different test that handles when options are set via options={"something": 1}.